build-logic/build.gradle.kts

```diff
@@ -0,0 +1,8 @@
+plugins {
+    `embedded-kotlin` apply false
+}
+
+allprojects {
+    group = "ch.tutteli.atrium"
+    version = "1.0.0"
```

Comment:
@vlsi is there a way that we need to define those only once and not here and in /build.gradle.kts?

Reply:
This is a different version. This is a version for the "build logic project" which you probably never need to update.

Reply:
However, the version number here is mostly for aesthetics purposes. Build logic projects are not typically used in their "group:artifact:version" notation, and I guess the only usage is `api("ch.tutteli.atrium.build-logic:gradle-plugin")` which does not really require a version since Gradle uses substitution from the included build.

It probably makes sense to update group to something like `ch.tutteli.atrium.build-logic` here to avoid confusion with main artifacts.